### ISSUE TITLE
None Type Error

### ISSUE THREAD

**Issue by Chetan11-dev** [daijro/hrequests#1]
Thanks for Library, 

Faced this error in Windows 
```
  File "C:\Users\Chetan\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.9_qbz5n2kfra8p0\LocalCache\local-packages\Python39\site-packages\hrequests\response.py", line 127, in Response
    elapsed: timedelta | None = None
TypeError: unsupported operand type(s) for |: 'type' and 'NoneType'
```
Fix was to set `elapsed = None`. Please add it to library to help people.

**Comment by daijro** [comment]
Fixed in v0.4.2

**Comment by Chetan11-dev** [comment]
Thanks
